### PR TITLE
W6-project-survey-vite-Eliane&Yia

### PR DESCRIPTION
**Pull request by Citronskal**
## Netlify link
https://survey-bedtime-story.netlify.app/

## Collaborators

[Citronskal, El1an3]

### CODE REVIEW

File: src/assets/components/Arrays.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  {
+    name: "Dreamy Drawing",
+    description:
+      "Under the soft glow of starlight your canvases become a portal to a fantastical world. You dip your brushes into pots of liquid colors. And when the final touches are made and the last strokes of paint are applied, you step back to admire your masterpieces, knowing that you have created something truly magical.",
+  },
+  {
+    name: "Night of Stargazing",
+    description:
+      "You set up a cozy spot. You gaze at the stars, identify constellations and share stories about the wonders of the universe. Curious about what might be out there in outer space, you slowly fall asleep.",
+  },
```
Comment (trista1987) [inline src/assets/components/Arrays.jsx:46]: It looks so clean on this page. good to see!

File: src/assets/components/Charactername.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// eslint-disable-next-line react/prop-types
+export function Charactername({ value, onChange }) {
+  return (
+    <div className="elementContainer">
+      <label htmlFor="characterName">
+        <h2 className="question">What is your character&#39;s name?</h2>
+        <input
+          aria-describedby="characterName"
```
Comment (trista1987) [inline src/assets/components/Charactername.jsx:8]: Good job for the accessibility!

Comment (trista1987) [reply src/assets/components/Charactername.jsx:8]: `aria-describedby="characterName"` we think here you should use kebab case

File: src/assets/components/Header.jsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+export const Header = () => {
+  return (
+    <header>
+      <h1>Create your own bedtime story</h1>
+      <img
+        src="/bedtimeheader.webp"
+        id="headerImg"
+        alt="Image of a flying book in a library "
+      ></img>
```
Comment (trista1987) [inline src/assets/components/Header.jsx:9]: Do we need `</img>`?

File: src/assets/components/Questions.jsx [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+  function handleSubmit(event) {
+    console.log("Form submitted");
+
+    event.preventDefault();
+    if (
+      charactername !== "" &&
+      location !== "" &&
+      companion !== "" &&
+      adventure !== ""
```
Comment (trista1987) [inline src/assets/components/Questions.jsx:24]: nice validation!

File: src/assets/components/Questions.jsx [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+                {
+                  adventuresArray.find(
+                    (adventureElement) => adventureElement.name === adventure
+                  )?.description
+                }
+              </p>
+            )}
+          </div>
+          <button className="submitButton" onClick={makeAnotherStory}>
+            Make another story
```
Comment (trista1987) [inline src/assets/components/Questions.jsx:83]: Could you use the SubmitButton component for this as well?

File: src/index.css [hunk 6]
```diff
@@ -1,13 +1,204 @@
...
+  width: 100%;
+  min-height: 200px;
+  max-height: 300px;
+  object-fit: cover;
+}
+
+.questionForm,
+.elementContainer {
+  margin: 0.75em 0;
+  /* width: 100%; */
```
Comment (trista1987) [inline src/index.css:45]: remember to remove your comment

File: src/index.css [hunk 7]
```diff
@@ -1,13 +1,204 @@
...
+}
+
+.appContainer {
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+}
+
+.formContainer {
+  min-height: calc(100% - 30px);
```
Comment (trista1987) [inline src/index.css:139]: Smart way for using `calc`!

**Review by trista1987 — COMMENTED** [review]
You girls did a really good job! Very impressive! The survey works well. But we think for the classname you should use kebab case.

**Review by AntonellaMorittu — APPROVED** [review]
Hello Eliane & Yia, congrats with your Survey project! The app satisfies the requirements and you chose a nice personal theme ⭐  Well done with responsiveness and accessibility, you did a great job working with the state, keep up the good work! 👏

**Pull request closed without merging**